packages/client/modules/userDashboard/components/GcalModal/VideoConferencing.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from '@emotion/styled'
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown'
+import {Close} from '@mui/icons-material'
+import React from 'react'
+import {MenuPosition} from '../../../../hooks/useCoords'
+import useMenu from '../../../../hooks/useMenu'
+import VideoConferencingMenu from './VideoConferencingMenu'
+import RaisedButton from '../../../../components/RaisedButton'
+import {PALETTE} from '../../../../styles/paletteV3'
+import {Elevation} from '../../../../styles/elevation'
+import GoogleMeetProviderLogo from '../../../../components/GoogleMeetProviderLogo'
+import ZoomProviderLogo from '../../../../components/ZoomProviderLogo'
+
+const StyledButton = styled(RaisedButton)({
```

Comment:
+1 just curious here, why use emotion here for wrapping emotion components vs. using tailwinds?